index.html

```diff
@@ -889,67 +889,133 @@ <h3>Supporting algorithms</h3>
           </li>
         </ul>
       </aside>
-      The <dfn>passes privacy test</dfn> steps given the argument |observer:PressureObserver| and
-      its [=relevant global object=] |relevantGlobal|, are as follows:
-      <ol>
-        <li>
-          If |relevantGlobal| is a {{WorkerGlobalScope}} object:
-          <ol>
-            <li>
-              If |relevantGlobal|'s relevant worker is not a <a href="https://html.spec.whatwg.org/multipage/workers.html#active-needed-worker">
-              active needed worker</a>, return false.
-            </li>
-            <li>
-              Otherwise, return true.
-            </li>
-          </ol>
-        </li>
-        <li>
-          If |relevantGlobal| is a {{Window}} object:
-          <ol>
-            <li>
+      <p>
+        To determine the <dfn>owning global object</dfn> for a {{DedicatedWorkerGlobalScope}} |workerGlobalScope|:
```

Comment:
Although the [Web IDL declaration in Chromium](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/workers/worker.idl;l=32;drc=7bb4923c49b3d122cae306abf81034ea25f4aa4d) is

```webidl
[Exposed=(Window,DedicatedWorker)]
interface Worker : EventTarget {
  // ...
};
```

[the spec](https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-worker-interface) has

```webidl
[Exposed=(Window,DedicatedWorker,SharedWorker)]
interface Worker : EventTarget {
  // ...
};
```

so you need to deal with the shared worker case too I guess?

Reply:
Shared workers can only be instantiated from Window so they will only have one owning global object (Window). There is a note about it in this PR.

The above algo is very similar to the one in page life cycle

Reply:
> Shared workers can only be instantiated from Window so they will only have one owning global object (Window)

Yeah, but these steps do not seem to cover the Document -> Shared worker -> Worker case. That Worker's owner set does not consist of either a Document or a DedicatedWorkerGlobalScope, right?

Reply:
[Global=(Worker,DedicatedWorker),Exposed=DedicatedWorker]
interface DedicatedWorkerGlobalScope

You cannot create a dedicated worker from a shared one

Reply:
That's the global scope, not the interface -- by that logic one wouldn't be able to create a dedicated worker from a window either.

Looking at the interface IDL, it does look possible to create a dedicated worker from a shared one (but not in Chromium).

Reply:
Added new commit to fix this